docs/src/assets/js/themes.js

```diff
@@ -8,11 +8,6 @@
         btn.setAttribute("aria-pressed", "false");
     }
 
-
-    let theme = window.localStorage.getItem("theme");
-    document.documentElement.setAttribute('data-theme', theme);
-    if (!theme) document.documentElement.setAttribute('data-theme', 'light');
-
```

Comment:
We are already setting the theme [here](https://github.com/eslint/eslint/blob/10249ad1a961463b6b347be71c074951ab8e2652/docs/src/_includes/layouts/base.njk#L48-L56) so setting it again in this script does seem unnecessary (and this code wasn't checking `prefers-color-scheme`).

But, we have some styles for [`html[data-theme="light"]`](https://github.com/eslint/eslint/blob/10249ad1a961463b6b347be71c074951ab8e2652/docs/src/assets/scss/tokens/themes.scss#L122), and now we're removing the code that sets it as the default, so maybe we should update [this code](https://github.com/eslint/eslint/blob/10249ad1a961463b6b347be71c074951ab8e2652/docs/src/_includes/layouts/base.njk#L48-L56) to set `"light"` as the default, like in the script on new.eslint.org ([here](https://github.com/eslint/new.eslint.org/blob/d1ff91e44fcd7814b355080a94d3545d2cad1ed0/src/_includes/layouts/base.html#L67-L75)), or set is direcly on the html tag (which could be better for users who have disabled JavaScript).

Reply:
https://github.com/eslint/eslint/blob/10249ad1a961463b6b347be71c074951ab8e2652/docs/src/assets/js/themes.js#L25-L28

We can also remove [these checks](https://github.com/eslint/eslint/blob/10249ad1a961463b6b347be71c074951ab8e2652/docs/src/assets/js/themes.js#L25-L28), right?

Reply:
Looking into them today! :)